examples/suit_update/Makefile

```diff
@@ -82,6 +85,9 @@ TEST_EXTRA_FILES += $(SLOT_RIOT_ELFS) $(SUIT_SEC) $(SUIT_PUB)
 
 include $(RIOTBASE)/Makefile.include
 
+# Export default path to fetch an update
+CFLAGS += -DSUIT_MANIFEST_RESOURCE=\"$(SUIT_COAP_ROOT)/$(SUIT_NOTIFY_MANIFEST)\"
```

Comment:
This change is quite interesting. I'm thinking that an improvement of the new `suit` shell command could also be based on that: calling the `suit` without parameter would trigger the update based on the pre-provisionned manifest resource. What do you think ?
Not needed for this PR though.

Reply:
Another minor code comment: this line could be moved before include `$(RIOTBASE)/Makefile.include`

Reply:
How about I move it next to `peripg_gpio_irq`:
```
# Optional feature to trigger suit update through gpio callback
FEATURES_OPTIONAL += periph_gpio_irq
# Default COAP manifest resource location when fetched though gpio callback
CFLAGS += -DSUIT_MANIFEST_RESOURCE=\"$(SUIT_COAP_ROOT)/$(SUIT_NOTIFY_MANIFEST)\"
```

Reply:
With a blank line between the 2

Reply:
Ups. No it actually can't be moved higher since it depends on the value of `SUIT_COAP_ROOT` and `SUIT_NOTIFY_MANIFEST`.